phpcs.xml.dist

```diff
@@ -6,6 +6,15 @@
   <arg name="extensions" value="inc,install,module,php,profile,test,theme,yml"/>
 
   <rule ref="Drupal"></rule>
+  <rule ref="DrupalPractice">
+  <!-- Here Variables would have had to been removed-->
+    <exclude-pattern>src/Plugin/GraphQL/DataProducer/DataProducerProxy.php</exclude-pattern>
```

Comment:
we should exclude them for the specific sniff, not for DrupalPractice in general. Like with the sniffs in the Drupal standard.